examples/server.c

```diff
@@ -116,12 +116,20 @@ outargMethod(UA_Server *server,
 int main(int argc, char** argv) {
     signal(SIGINT, stopHandler); /* catches ctrl-c */
 
-    UA_ByteString certificate = loadCertificate();
-    UA_ServerConfig *config =
-        UA_ServerConfig_new_minimal(4840, &certificate);
-    UA_ByteString_deleteMembers(&certificate);
+    UA_ByteString cert = loadCertificate();
+
+    UA_ServerConfig *config = UA_ServerConfig_new_minimal(16664, &cert);
+    if(config == NULL)
```

Comment:
The `cert` variable can already be deleted here, since it is copied in `UA_ServerConfig_new_minimal` (as it is already in master).
This also avoids memleaks on the `return -1` cases